changelog.md

```diff
@@ -6,6 +6,43 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres
 to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+### 0.41.0
+
+### Added
+
+* `state init` is now a stable command, meaning you no longer need to opt-in to
+  unstable commands to use it.
+* Signing up for a new account now opens the account creation page in your
+  browser. Instead of bringing you to the sign in page.
+* `state shell` can now detect currently active subshells preventing nested
+  shells from being created.
+* The State Tool will now print a message when starting a subshell, indicating
+  the project that is being used.
+* Wildcard and partial version matching is now supported for `state install` 
+  and for language versions with `state init`.
+  For example: 
+  * `state install pytest@2.x`
+  * `state init "python@>3.7,<3.8"`
```

Comment:
I don't think this one is new? Only the wildcards. You can already do the larger than / smaller than on the currently released version.

Reply:
This example is from this story: https://activestatef.atlassian.net/browse/DX-2010. It looks like there was something preventing us from doing this before?

Reply:
It had to do with what partial versions got turned into. But fully qualified versions like the second example were already working.

The change is basically that you now have to use the wildcard notation if you don't want to specify a full version or use the larger than/smaller than notation.